clang/test/CodeCompletion/cpp23-explicit-object.cpp

```diff
@@ -42,7 +42,23 @@ int func3() {
 
 int func4() {
   // TODO (&A::foo)(
-  (&A::bar)(
+  (&A::bar)()
 }
```

Comment:
Had to add this paren, otherwise compilation fails

```
clang/test/CodeCompletion/cpp23-explicit-object.cpp:46:1: error: expected expression
   46 | }
      | ^
1 error generated.
```